lms/services/lti_grading/_v13.py

```diff
@@ -23,19 +25,31 @@ class LTI13GradingService(LTIGradingService):
     ]
 
     def __init__(
-        self, line_item_url, line_item_container_url, ltia_service: LTIAHTTPService
+        self,
+        line_item_url,
+        line_item_container_url,
+        ltia_service: LTIAHTTPService,
+        product_family: Family,
     ):
         super().__init__(line_item_url, line_item_container_url)
         self._ltia_service = ltia_service
+        self._product_family = product_family
 
     def read_result(self, grading_id) -> GradingResult:
         result = GradingResult(score=None, comment=None)
+
+        params = {"user_id": grading_id}
+        if self._product_family == Family.BLACKBOARD:
```

Comment:
As discussed we usually do this kind of conditional as part of the product implementation.

I don't think that would be the best decision here. We'll have to create a weird abstraction about either this whole method or just part of it (which params we send? any filtering after getting the line items?)

I reckon the conditionals here are more clear than the alternative we hope BB fixed this and we can just remove the conditionals.

If over time this service grows a lot of more conditionals we can decide which abstraction to build then.